moped-database/migrations/1696622988854_add_feature_length_column/up.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE feature_street_segments
+    ADD COLUMN length_feet INTEGER GENERATED always AS (
+Round(ST_Length(geography) * 3.28084)) stored;
+
+ALTER TABLE feature_drawn_lines
+    ADD COLUMN length_feet INTEGER GENERATED always AS (
+Round(ST_Length(geography) * 3.28084)) stored;
```

Comment:
i tested this migration locally against production data and it is lightening fast. we should have no concerns with a migration timeout when we deploy it to production.